### ISSUE TITLE
[HW1] Password Confirmation requirement

### ISSUE THREAD

**Issue by yuyupopo** [swsnu/swppfall2017#25]
In the requirements, it says " Password Confirmation: Must match password "
1. Then, if both the password and password_confirmation form is empty, should the program decide password confirmation as correct?
2. Should password_confirmation should be seen correct as long as it matches password, but the password itself is incorrect?

Thanks

**Comment by wonook** [comment]
Yes, `password_confirmation` should simply depend on the `password` form. Since `password` form checks for correctness, if it doesn't pass with the `password` form, the `password_confirmation` wouldn't need to check again.
